pkg/sfu/buffer/rtpstats_receiver.go

```diff
@@ -205,7 +217,19 @@ func (r *RTPStatsReceiver) Update(
 		flowState.ExtTimestamp = resTS.ExtendedVal
 	} else { // in-order
 		if gapSN >= cNumSequenceNumbers {
-			r.logger.Warnw("large sequence number gap", nil, "prev", resSN.PreExtendedHighest, "curr", resSN.ExtendedVal, "gap", gapSN)
+			r.logger.Warnw(
```

Comment:
This is still happening on subscriber side. Not able to see it how. Seems to happen across relay. Trying to log a bit more to check if there are more clues. I think this is going to be hard to figure out without more logging, but trying. This is not happening in `preprod`. So, has been hard to figure out.